src/Microsoft.Data.Sqlite/Strings.resx

```diff
@@ -186,4 +186,7 @@
   <data name="DefaultNativeError" xml:space="preserve">
     <value>For more information on this error code see https://www.sqlite.org/rescode.html</value>
   </data>
+  <data name="AmbiguousParameterName" xml:space="preserve">
+    <value>Cannot bind the value for parameter '{parameterName}' because multiple matching parameters were found in the command text.</value>
```

Comment:
Tell how to fix it. Something like: Specify which one to bind to by including the prefix in the parameter name (e.g. "@{parameterName}").